src/main/java/mate/academy/Main.java

```diff
@@ -10,12 +10,12 @@
 public class Main {
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = new User("bill", "qwerty");
```

Comment:
What is the reason to make this change?

Reply:
I thoght it could be default values if user didn't login